Library/Formula/openssl.rb

```diff
@@ -120,10 +133,8 @@ def post_install
     openssldir.mkpath
     (openssldir/"cert.pem").atomic_write `security find-certificate -a -p #{keychains.join(" ")}`
 
-    if MacOS.version == :yosemite
-      (openssldir/"certs").install resource("Equifax_CA")
-      system bin/"c_rehash"
-    end
+    # Remove this once 1.0.2b lands.
+    rm_f openssldir/"certs/Equifax_CA" if MacOS.version == :yosemite
```

Comment:
The file was named `578d5c04.0` on my system as well, although I was using the [mozilla ca bundle from the curl website](http://curl.haxx.se/ca/cacert.pem), so I think it's not safe to simply remove the symlink. A better solution would be to run `system bin/"c_rehash"` after the file is removed, because it will clean up orphaned symlinks.